### PR TITLE
Update Issuing API preview

### PR DESCRIPTION
**Pull request by cko-devops**

### CODE REVIEW

File: nas_spec/paths/issuing/cards@{cardId}@credentials.yaml [hunk 1]
```diff
@@ -1,39 +1,38 @@
...
+    required: false
+    schema:
+      type: string
+      enum:
+      - number
+      - cvc2
+      example: number,cvc2
+      description: "One or more comma-separated credential fields. \nThis works like\
+        \ a logical *AND* operator and returns the combined credentials in the response.\n\
+        Defaults to no credentials being returned.\n"
```
Comment (colm-prunty-cko) [inline nas_spec/paths/issuing/cards@{cardId}@credentials.yaml:28]: This looks ok in the preview but I'm not sure why there are `"` and `\n\` all around this one

File: nas_spec/paths/issuing/cards@{cardId}@3ds-enrollment.yaml [hunk 2]
```diff
@@ -41,23 +51,22 @@ post:
...
     '500':
       description: Internal Server Error
-
 patch:
   tags:
-    - Cards
+  - Cards
   summary: Update already 3DS enrolled card
-  description:
+  description: null
```
Comment (colm-prunty-cko) [inline nas_spec/paths/issuing/cards@{cardId}@3ds-enrollment.yaml:58]: Looks like this doesn't actually write "null" as the description on the page

File: nas_spec/paths/issuing/cards@{cardId}@credentials.yaml (hunk 1 above)
Comment (badr-kedjour-cko) [reply nas_spec/paths/issuing/cards@{cardId}@credentials.yaml:28]: If you look at the [original](https://github.com/cko-issuing/openapi/blob/main/spec/paths/card-management/cards%40%7BcardId%7D%40credentials.yaml), its using the [literal style](https://yaml.org/spec/1.2-old/spec.html#style/block/literal) which will translate to the string that you are seeing here. My guess is the [library](https://pypi.org/project/PyYAML/) that we are using kept only the translated string.

File: nas_spec/paths/issuing/cards@{cardId}@3ds-enrollment.yaml (hunk 2 above)
Comment (badr-kedjour-cko) [reply nas_spec/paths/issuing/cards@{cardId}@3ds-enrollment.yaml:58]: This is expected behavior

**Review by fabio-corneti-cko — APPROVED** [review]

**Pull request merged**